msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+| `m365_selfserviceappointmentid` | StringType | Additional tracking ID for the appointment, if the appointment has been created directly by the customer on the scheduling page, as opposed to by a staff member on the behalf of the customer.|
+| `m365_serviceid` | StringType | The ID of the bookingService associated with this appointment.|
+| `m365_servicelocation` | StringType | The location where the service is delivered. {\"address\":{\"city\":\"\",\"countryOrRegion\":\"\",\"postalCode\":\"\",\"postOfficeBox\",\"state\":\"\",\"street\":\"\",\"type\" },\"coordinates\":{\"accuracy\",\"altitude\",\"altitudeAccuracy\",\"latitude\",\"longitude\" },\"displayName\":\"Our office address\",\"locationEmailAddress\",\"locationType\",\"locationUri\":\"\",\"uniqueID\",\"uniqueIDType\" } |
+| `m365_servicename` | StringType | The name of the bookingService associated with this appointment. This property is optional when creating a new appointment. If not specified, it's computed from the service associated with the appointment by the serviceID property. |
+| `m365_servicenotes` |StringType | Notes from a bookingStaffMember. The value of this property is available only when reading this bookingAppointment by its ID.|
+| `m365_smsnotificationsenabled` | BooleanType | True indicates SMS notifications will be sent to the customers for the appointment. Default value is false.|
+| `m365_staffmemberids` | StringType | The ID of each bookingStaffMember who is scheduled in this appointment. Stored as a comma separated string.[\”string\”] |
+| `m365_startdatetime` | DateTimeType | The date, time, and time zone that the appointment begins.|
+
+## Graph Drive
+
+* Entity name: m365_graphdrive
+* Graph resource: [drive resource type](/graph/api/resources/drive)
+* Sorting isn't supported.
+* Filtering isn't supported.
+* Server driven pagination is supported.
+
+### Attributes for Graph Drive
+
+|Column |Dataverse Type |Details |
+|---|---|---|
+|`m365_collaborationrootid` |StringType | Collaboration root ID of the collaboration session the record is associated with. If the record is associated with multiple collaboration sessions this will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records. |
+|`m365_createdby` |StringType |Identity of the user, device, or application that created the item. Read-only. This attribute is a JSON encoded string for example { "user": { "displayName": "System Account" } } |
+|`m365_createddatetime` |DateTimeType |Date and time of item creation. Read-only. |
+|`m365_description` |StringType |Provide a user-visible description of the drive. Read-only. |
+|`m365_drivetype` |StringType |Describes the type of drive represented by this resource. OneDrive personal drives will return personal. OneDrive for Business will return business. SharePoint document libraries will return documentLibrary. Read-only. |
+|`m365_graphdriveid` |GuID |Unique Identifier of the graph drive. |
```

Comment:
Change to

| `m365_graphdriveId` | guid | Unique Identifier of the graph drive. |